dev/com.ibm.ws.wsat.common.3.2/.classpath

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <classpath>
-	<classpathentry kind="src" path="com.ibm.ws.wsat.common-src"/>
```

Comment:
Another project where you move the source instead of just updating the bnd.bnd file and changing the feature files.